boot/freeldr/freeldr/include/arch/i386/i386.h

```diff
@@ -29,7 +29,9 @@
 
 /* Bits to shift to convert a Virtual Address into an Offset in the Page Directory */
 #define PDE_SHIFT 22
-#define PDE_SHIFT_PAE 18
+
+#define PDE_PER_PAGE (PAGE_SIZE / sizeof(HARDWARE_PDE_X86)) // 0x400 (1024)
```

Comment:
It was added with the PAE_PTE_PER_PAGE from my header file.
In general, I did not plan to refactoring Non-PAE code, it seems to me it would be more appropriate in another PR.
I suggest to remove it.